engine/language-server/src/main/java/org/enso/languageserver/boot/resource/InitializationComponentInitialized.java

```diff
@@ -0,0 +1,19 @@
+package org.enso.languageserver.boot.resource;
+
+/** Object indicating that the initialization is complete. */
+public final class InitializationComponentInitialized {
+
+  private static final class InstanceHolder {
```

Comment:
There is no need for `InstanceHolder` when the only method and `INSTANCE` field of the `InitializationComponentInitialized` class deal with holding and initializing the instance.